src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -40,7 +41,8 @@
         </ul>
     </div>
 </div>
-<div class="col-sm-12 margin-bottom-15px background-color-light-green">
+<!-- Fix for collapsing margin problem. Reference: http://stackoverflow.com/questions/6204670 -->
```

Comment:
hey if I am not wrong this is one of the issues for ensuring mobile friendliness / UI in general. Since this is related to your FYP, do you think this "guideline" can be documented in more detail? ie as a technical report? 
For instance for each of the issue: 
1. what the problem is
2. possible solutions
3. why the solution you suggest

You can set the standard and bounce it by the prof